### ISSUE TITLE
Improvement of the "Repeat mode" and "Shuffle mode" buttons.

### ISSUE THREAD

**Issue by SokoloffA** [losnoco/Cog#177]
The differences of icons on the "Repeat Mode" button are too small and user have to guess what they mean. I suggest changing the style of the buttons to drop-down buttons, as in "Grouping" button in the finder. Then the buttons will have signatures and their purpose will become clear.

![Button](https://user-images.githubusercontent.com/854935/140486253-731cd565-0dbf-48bf-a340-f4628185a451.png) 

![Menu](https://user-images.githubusercontent.com/854935/140486409-a6e24723-5b19-4ce3-858b-5b6fb20540ed.png)

It's the same with the "Shuffle mode" button.

**Comment by nevack** [comment]
Yeah, icons are not that desciptive, for now I suggest to use Menu Bar's Playlist submenu
![image](https://user-images.githubusercontent.com/8330119/140517809-bf304814-7445-4038-a2b3-048f217a59f0.png)
